wiki/FAQ.md

```diff
@@ -13,8 +13,29 @@ Unsoluble has a great video guide available here https://www.youtube.com/watch?v
 ## How to create Spell Scrolls
 There are two ways to create a Spell Scroll from a spell, either by using the Right Click context menu and using the "Create Scroll" option or by dragging a Spell onto an Actor's Inventory tab. A dialog will be displayed with options as to what description should be shown and what levels the spell should be created at (defaults to the spell's base level). A reference to the spell scroll rules or their entire text from the SRD rules will be added to the spell's description.
 
-## What are the plans for the upcoming 2024 rulebook releases?
-In September Foundry will be publishing an official premium module for the updated Player's Handbook. Until the revised SRD is published in 2025 this will be the only way to access pre-made content from the new rulebooks in the system. Once the revised SRD is made available the content will be added to the system in the same way the SRD 5.1 content is currently available.
+## What are the plans for the 2024 rulebook releases?
+In Foundry publishes a official premium modules for the updated [Player's Handbook](http://foundryvtt.com/packages/dnd-players-handbook) and [Dungeon Master's Guide](https://foundryvtt.com/packages/dnd-dungeon-masters-guide), and the upcoming [Monster Manual](https://foundryvtt.com/packages/dnd-monster-manual) is available for pre-order. Until the revised SRD is published in 2025 this will be the only way to access pre-made content from the new rulebooks in the system. Once the revised SRD is made available the content will be added to the system in the same way the SRD 5.1 content is currently available.
```

Comment:
```suggestion
Foundry published an official premium module for the updated [Player's Handbook](http://foundryvtt.com/packages/dnd-players-handbook) and [Dungeon Master's Guide](https://foundryvtt.com/packages/dnd-dungeon-masters-guide), and the upcoming [Monster Manual](https://foundryvtt.com/packages/dnd-monster-manual) is available for pre-order. Until the revised SRD is published in 2025 this will be the only way to access pre-made content from the new rulebooks in the system. Once the revised SRD is made available, the content will be added to the system in the same way the SRD 5.1 content is currently available.
```

Reply:
So it's settled that the system will package both the legacy and new SRD?

Reply:
Nope, didn't touch that language.